react/components/CreateOrganizationButton/CreateOrganizationButton.tsx

```diff
@@ -55,11 +56,13 @@ const CreateOrganizationButton = () => {
           icon={<IconPencil />}
           onClick={() => setOpen(true)}
         />
-        <MenuItem
-          label={formatMessage(messages.addBulk)}
-          icon={<IconCloudArrowUp />}
-          onClick={() => setUploadModalOpen(true)}
-        />
+        {!!data && (
```

Comment:
Just to be sure: even when there are no ongoing imports, will this data object be defined? Just making sure that this dropdown item won't disappear on the first usage for an account, for example.

Reply:
Yes, when we don't have imports is returned an empty array, and `!![]` results true